bank_account_project.py

```diff
@@ -1,169 +0,0 @@
-'''
-***All credit for the data sourced for this project goes to exchangerates.org.uk***
-
-This program simulates a transaction with an ATM machine. If the user decides to 
-withdraw money from their bank account, they are given the option to enter the 
-withdrawal amount in a foreign currency. The program will grab the most updated 
-currency exchange figures and return the amount remaining in the bank account 
-in GBP. 
-'''
-
-#This function scrapes conversion rates between GBP(£) and some of the most 
-#common currencies in the world. The function takes one argument, gbp, that 
-#must be either an integar or a float. 
-def currency_func(gbp):
-
-    import requests
```

Comment:
Move imports to the top of the file (this is convention, and prevents the same thing from being imported multiple times)